ddtrace/llmobs/_integrations/openai.py

```diff
@@ -184,11 +184,8 @@ def _extract_llmobs_metrics_tags(
                 metrics[CACHE_READ_INPUT_TOKENS_METRIC_KEY] = cached_tokens
             return metrics
         elif kwargs.get("stream") and resp is not None:
-            model_name = span.get_tag("openai.response.model") or kwargs.get("model", "")
-            _, prompt_tokens = _compute_prompt_tokens(
-                model_name, kwargs.get("prompt", None), kwargs.get("messages", None)
-            )
-            _, completion_tokens = _compute_completion_tokens(resp, model_name)
+            _, prompt_tokens = _compute_prompt_tokens(kwargs.get("prompt", None), kwargs.get("messages", None))
+            _, completion_tokens = _compute_completion_tokens(resp)
```

Comment:
looks like for both of these functions we can get rid of the first return argument since they're not used